src/components/page-layout/page-layout.js

```diff
@@ -38,32 +38,47 @@ export default class PageLayout extends React.PureComponent {
 
   // render the page's content
   renderContent = (config, parentPath, parent, hasSection) => {
-    const { constants, frontMatter, location, domain } = this.props;
+    const { constants, frontMatter, location, domain, navigation } = this.props;
+    // If multi-structured show section name
```

Comment:
Note to self: try using `switchedNavigation` in place of `navigation` here to minimize complexity of `getSubPages` function in page layout `utils.js`.

Reply:
I wasn't able to figure out how to use `switchedNavigation` because it doesn't include `hierarchy`, which is used `getSubPages`. Let me know if I'm missing something!